### PR TITLE
Changing the terraform module reference

### PR DESCRIPTION
**Pull request by sylviamclaughlin**
# Summary | Résumé

Changing the terraform module reference link to be consistent with the terraform documentation - https://developer.hashicorp.com/terraform/language/modules/sources#modules-in-package-sub-directories

### CODE REVIEW

**Review by gcharest — DISMISSED** [review]

**Review by patheard — CHANGES_REQUESTED** [review]
Could you ping me once you've merged the TF module fix (thanks for that btw).  I want to double-check the plan once it re-runs since this repo touches all our Staging and Prod accounts so it makes me nervous.

File: bootstrap/central_account_iam/main.tf [hunk 1]
```diff
@@ -4,7 +4,7 @@ locals {
 
 # Role used by Terraform to manage all satellite accounts
 module "gh_oidc_roles" {
-  source = "github.com/cds-snc/terraform-modules?ref=v3.0.18//gh_oidc_role"
+  source = "github.com/cds-snc/terraform-modules//gh_oidc_role?ref=v6.1.4"
```
Comment (patheard) [inline bootstrap/central_account_iam/main.tf:7]: Could you please revert this one back to the original version.  There was a breaking change that came in at `v4.0.0` of the module and it'll be hard to fix:
```suggestion
  source = "github.com/cds-snc/terraform-modules//gh_oidc_role?ref=v3.0.18"
```

File: terragrunt/aws/central_account/main.tf [hunk 2]
```diff
@@ -8,7 +8,7 @@ locals {
 
 # Role used by Terraform to manage all satellite accounts
 module "gh_oidc_roles" {
-  source = "github.com/cds-snc/terraform-modules?ref=v3.0.19//gh_oidc_role"
+  source = "github.com/cds-snc/terraform-modules//gh_oidc_role?ref=v6.1.4"
```
Comment (patheard) [inline terragrunt/aws/central_account/main.tf:11]: Same comment about the version revert:
```suggestion
  source = "github.com/cds-snc/terraform-modules//gh_oidc_role?ref=v3.0.19"
```

File: bootstrap/central_account_iam/main.tf (hunk 1 above)
Comment (sylviamclaughlin) [reply bootstrap/central_account_iam/main.tf]: This has been updated! Thank you Pat for your suggestion as I would have broken everything 😬

File: terragrunt/aws/central_account/main.tf (hunk 2 above)
Comment (sylviamclaughlin) [reply terragrunt/aws/central_account/main.tf]: This has been updated!

File: terragrunt/aws/central_account/main.tf [hunk 3]
```diff
@@ -8,7 +8,7 @@ locals {
 
 # Role used by Terraform to manage all satellite accounts
 module "gh_oidc_roles" {
-  source = "github.com/cds-snc/terraform-modules?ref=v3.0.19//gh_oidc_role"
+  source = "github.com/cds-snc/terraform-modules//gh_oidc_role?ref=v3.0.1"
```
Comment (patheard) [inline terragrunt/aws/central_account/main.tf:11]: Could you just add back the missing `9` (I may have accidentally deleted it in my last comment suggestion):
```suggestion
  source = "github.com/cds-snc/terraform-modules//gh_oidc_role?ref=v3.0.19"
```

**Comment by sylviamclaughlin** [comment]
Sorry @patheard - I don't think it was you stripping the 9. I most likely did it. It has been corrected now.

**Pull request merged**